AGENTS.md

```
@@ -0,0 +1,19 @@
+# Instructions for AI Agents
+
+## 1. Build Safety
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)